client/webserver/webserver.go

```diff
@@ -632,29 +635,20 @@ func New(cfg *Config) (*WebServer, error) {
 // fetchLatestVersion is a helper function to retrieve the latest version of the app
 // from github.
 func (w *WebServer) fetchLatestVersion(ctx context.Context) {
-	ticker := time.NewTicker(1 * time.Minute)
-	defer ticker.Stop()
-
-	for {
-		select {
-		case <-ticker.C:
-			var response struct {
-				TagName string `json:"tag_name"`
-			}
-
-			err := dexnet.Get(ctx, "https://api.github.com/repos/decred/dcrdex/releases/latest", &response, dexnet.WithSizeLimit(1<<22))
-			if err != nil {
-				log.Debugf("Error getting latest version: %v", err)
-				continue
-			}
-
-			w.latestVersion = strings.Trim(response.TagName, "v")
-			return
+	var response struct {
+		TagName string `json:"tag_name"`
+	}
 
-		case <-ctx.Done():
-			return
-		}
+	err := dexnet.Get(ctx, "https://api.github.com/repos/decred/dcrdex/releases/latest", &response, dexnet.WithSizeLimit(1<<22))
+	if err != nil {
+		log.Debugf("Error getting latest version: %v", err)
+		return
 	}
+
+	lastestVersion := latestVersionRegex.FindString(response.TagName)
+	appSemVersion := userAppVersion(w.appVersion, true)
+	w.newAppVersionAvailable = lastestVersion > appSemVersion
+	return
```

Comment:
This is still comparing strings... does that always work?

Reply:
> This is still comparing strings... does that always work?

Yes, it always works.

But if you insist, I can compare each individual number.

Reply:
> if (majorWeb < majorLocal || minorWeb < minorLocal || patchWeb <= patchLocal)

I used a more elaborate and bug-free check in https://github.com/decred/dcrdex/pull/3286/commits/256c35760a28857b5cfa256b2485a5c194694e9c.

IF majorWeb > majorLocal but minorWeb is < minorLocal, it'll will also cause a false positive like the string check.

Reply:
You are right..What I meant was IF majorLocal > MajorWeb, but minorWeb > minorLocal. At this point, I'm not assuming that the web version is always > the local version.